engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -106,6 +110,15 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
 
   private val calls = workflowDescriptor.backendDescriptor.workflowNamespace.workflow.calls
 
+  // TODO: We should probably create a trait which loads all the configuration (once per application), and let classes mix it in
+  // to avoid doing ConfigFactory.load() at multiple places
+  val MaxRetries = Try(ConfigFactory.load().getInt("system.max-retries")) match {
+    case Success(value) => value
+    case Failure(reason) =>
+      log.warning(s"Failed to load the max-retries value from the configuration. Defaulting back to a value of `10` : ${reason.getMessage}")
+      10
```

Comment:
Eeek! I'll secretly sneek `10` to the list of the ignored parameters for MagicNumber checks.